android/engine/src/test/java/org/smartregister/fhircore/engine/data/local/DefaultRepositoryTest.kt

```diff
@@ -564,7 +568,8 @@ class DefaultRepositoryTest : RobolectricTest() {
           sharedPreferencesHelper = mockk(),
           configurationRegistry = mockk(),
           configService = mockk(),
-          configRulesExecutor = mockk()
+          configRulesExecutor = mockk(),
```

Comment:
Why not use the injected `configRulesExecutor` as done on line `123` ?

Reply:
not my change, I only added a comma for next argument.